pkg/services/ngalert/migration/permissions.go

```diff
@@ -0,0 +1,300 @@
+package migration
```

Comment:
This is moved nearly 100% as-is from sqstore, not sure why it's not detected as a move. The only changes are to use the session through the context instead of through `folderHelper`. Browser diff: https://www.diffchecker.com/ldxQCY9E/